Basic-Car-Maintenance/Shared/Settings/Views/SettingsView.swift

```diff
@@ -23,6 +23,11 @@ struct SettingsView: View {
     @State private var errorDetails: Error?
     @State private var copiedAppVersion: Bool = false
     
+    @State private var selectedVehicleEvent: EditVehicleEvent?
+    @State private var isEditingVehicle = false
+    @State var editViewModel: EditVehicleView?
+    @State private var vehicleToEdit: Vehicle?
```

Comment:
You only need the `vehicleToEdit`, you shouldn't be using a separate model for editing, so `EditVehicleEvent` is not needed

Also don't call this `editViewModel` is not needed